### PR TITLE
chore: add scalar use toasts back in

### PR DESCRIPTION
**Pull request by antlio**
this pr re-introduces @scalar-usetoasts component + adds vue-sonner to it.

### CODE REVIEW

File: packages/use-toasts/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+```vue
+// ChildComponent.vue
+<script setup>
+import { useToasts } from '@scalar/use-toasts'
+
+const { toast } = useToasts()
+
+const sendMessage = () => {
+  toast('This is a message from the toaster!', 'success', { timeout: 2000 })
+}
```
Comment (geoffgscott) [inline packages/use-toasts/README.md:35]: Can you confirm this is the actual interface? I think it might be different with vue-sonner

Comment (antlio) [reply packages/use-toasts/README.md:35]: yes that's [how it's being used currently](https://github.com/scalar/scalar/blob/5c5bc45857082ddb2149cf949cc837ab3ad70ec1/packages/api-reference/src/hooks/useClipboard.ts)

**Comment by hwkr** [comment]
This looks good! I wouldn't mind adding some styles to make it look more like the other Scalar stuff. The old toast library had quite a bit of stuff like a loading ring and stuff.

**Review by hwkr — APPROVED** [review]
Lgtm!

File: packages/use-toasts/src/ScalarToast.test.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { describe, expect, it } from 'vitest'
+
+describe('example', () => {
```
[the remark this answers is not in the document]
Comment (antlio) [reply packages/use-toasts/src/ScalarToast.test.ts]: added!

**Pull request merged**